README.md

```diff
@@ -1,7 +1,5 @@
-[![Circle CI](https://circleci.com/gh/palantir/godel.svg?style=svg)](https://circleci.com/gh/palantir/godel)
+# gödel [![Circle CI](https://circleci.com/gh/palantir/godel.svg?style=svg)](https://circleci.com/gh/palantir/godel)
```

Comment:
Prefer style requested in previous feedback:

```
gödel		
=====

[![Circle CI](https://circleci.com/gh/palantir/godel.svg?style=svg)](https://circleci.com/gh/palantir/godel)

gödel is a Go...
```

Obviously just a matter of personal preference (plenty of high-profile public projects use different styles), but I visually prefer it to having the badges precede the header or be part of it. This is also the same as the style used by Kubernetes and feign for their README badges.